geoh5py/shared/utils.py

```diff
@@ -161,6 +161,22 @@ def merge_arrays(
     return np.r_[head, tail]
 
 
+def clear_array_attributes(entity: Entity, recursive: bool = False):
+    """
+    Clear all attributes from an entity.
```

Comment:
**suggestion (doc)**: clarify this is only a cache and that it does not affect the actual attribute values of the entity

Reply:
Done